tests/Base.t.sol

```diff
@@ -102,29 +114,39 @@ abstract contract Base_Test is Assertions, Constants, DeployOptimized, Modifiers
     //////////////////////////////////////////////////////////////////////////*/
 
     /// @dev Approves all contracts to spend tokens from the address passed.
-    function approveFactory(address from) internal {
+    function approveFactories(address from) internal {
         resetPrank({ msgSender: from });
-        dai.approve({ spender: address(merkleFactory), value: MAX_UINT256 });
+        dai.approve({ spender: address(merkleFactoryInstant), value: MAX_UINT256 });
+        dai.approve({ spender: address(merkleFactoryLL), value: MAX_UINT256 });
+        dai.approve({ spender: address(merkleFactoryLT), value: MAX_UINT256 });
+        dai.approve({ spender: address(merkleFactoryVCA), value: MAX_UINT256 });
     }
 
     /// @dev Generates a user, labels its address, funds it with test tokens, and approves the protocol contracts.
     function createUser(string memory name) internal returns (address payable) {
         address payable user = payable(makeAddr(name));
         vm.deal({ account: user, newBalance: 100 ether });
         deal({ token: address(dai), to: user, give: 1_000_000e18 });
-        approveFactory({ from: user });
+        approveFactories({ from: user });
         return user;
     }
 
-    /// @dev Deploys the Merkle Factory contract conditionally based on the test profile.
-    function deployMerkleFactoryConditionally() internal {
-        // Deploy the Merkle Factory.
+    /// @dev Deploys the Merkle Factories contracts conditionally based on the test profile.
```

Comment:
`Merkle Factory contracts` would be more apt similar to how you are using at other places